src/classes/CW_CampaignRewardTriggerHandler.cls

```diff
@@ -0,0 +1,30 @@
+public with sharing class CW_CampaignRewardTriggerHandler 
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.